app/models/concerns/waste_carriers_engine/can_have_registration_attributes.rb

```diff
@@ -124,6 +124,10 @@ def rejected_conviction_checks?
         conviction_sign_offs.last.rejected?
       end
 
+      def renewable?
```

Comment:
Why are we adding this here? Only 'registrations' can be renewed, so I'm not sure why it would make sense to also apply this to a renewing registration, and in future an edit registration.